### DISCUSSION TITLE
Github campus expert

### DISCUSSION THREAD

**Discussion by brunellaquaye** [community/community#130457] [GitHub Education] [answered]
### Hi everyone,

When can people register for GitHub campus expert?

**Comment by asdacosta** [comment]
February and August. 
https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/applying-to-be-a-github-campus-expert#:~:text=Applications%20to%20the%20program%20open,get%20to%20know%20the%20applicant.

**Comment by calyxish** [comment, marked as answer]
You can register for GitHub Campus Expert but there are certain criteria a person must fulfill before he will qualify for GitHub Campus Expert.
1.  Have the GitHub Student Developer Pack
2. Be 18 years of age or older
3. Be enrolled in a formal higher education institution
4. Have at least one year before graduating
5. Have had a GitHub account for at least six months

if you are interested, you can register in August 2024 or February  2025. But it is mainly done in February and August Every Year. 

FOR EXTENSIVE READING.
https://github.blog/2020-12-10-introducing-the-new-and-improved-campus-experts-program/

You can also subscribe to their NEWSLETTER.
https://github.blog/2020-12-10-introducing-the-new-and-improved-campus-experts-program/#newsletter
Thank you very much
